src/openforms/emails/tests/test_digest_functions.py

```diff
@@ -1361,6 +1361,74 @@ def test_invalid_logic_actions_with_exceptions_are_reported(self):
             invalid_logic_rules,
         )
 
+    @tag("gh-5271")
+    def test_invalid_logic_rules_with_reduce_and_accumulator_not_reported(self):
```

Comment:
```suggestion
    def test_valid_logic_rules_with_reduce_and_accumulator_not_reported(self):
```
the logic rule is valid, so this test name is confusing otherwise